web/src/app/procurement/all/page.tsx

```diff
@@ -13,6 +30,71 @@ const AllPage = () => {
 				columnSchema={columnSchema}
 				dataSource={dataSource}
 			/>
+
+			{/* Pending modal */}
+			<CustomModal
+				open={openPending}
+				title="Molave Youngs Milling Group"
+				width={"65%"}
+				titleStyle={{ fontSize: 20, fontWeight: "bold", color: "blue" }}
+				footerButtons={[
+					{
+						label: "Close",
+						color: "primary",
+						onClick: () => setOpenPending(false),
+					},
+				]}
+				handleClose={() => setOpenPending(false)}
+				content={modalContentPending}
+			/>
+
+			<CustomModal
+				open={openAddItems}
+				handleClose={() => setOpenAddItems(false)}
+				title="Add Items ..."
+				btnName={btnName}
+				width={600}
+				formProps={{
+					fields,
+				}}
+			/>
+
+			{/* Canvass modal */}
+			<CustomModal
+				open={openCanvass}
+				title="Molave Youngs Milling Group"
+				width={"65%"}
+				titleStyle={{ fontSize: 20, fontWeight: "bold", color: "blue" }}
+				footerButtons={[
+					{
+						label: "Close",
+						color: "primary",
+						onClick: () => setOpenCanvass(false),
+					},
+				]}
+				handleClose={() => setOpenCanvass(false)}
+				content={modalContentCanvass}
+			/>
+			<CustomModal
+				open={supplierModalOpen}
+				title="Molave Youngs Milling Group"
+				width={"65%"}
+				titleStyle={{ fontSize: 20, fontWeight: "bold", color: "blue" }}
+				footerButtons={[
+					{
+						label: "Close",
+						color: "primary",
+						onClick: () => setSupplierModalOpen(false),
```

Comment:
ibutang ni sa hook.ts din replace onClick actions

const handleToggleSupplierModal = () => setSupplierModal( (state) => !state )

```suggestion
						onClick: handleSupplierToggleModal ,
```